arm_diags/diags_all_multisites_for_cmip6.json

```diff
@@ -283,7 +283,23 @@
       "diags_set": "set9_aerosol_activation",
       "variables": ["cpc","ccn"],
       "sites": ["enac1"]
+                },
+          {
+      "diags_set": "set10_twolegged_metric",
+      "variables": ["Atmospheric Coupling Legs","Terrestrial Cooupling Legs"],
```

Comment:
No, they are not the actually variables names. These are not currently applied in the src codes. I would suggest changing this as follows to avoid any further confusion:

"variables": ["SH","LH","LCL","soil_moisture"]

The input data file include the time series of surface sensible and latent heat flux, lifting condensation level and soil moiture. Thank you!

Reply:
Both strings refer to the name of the metrics (not the variable names), which will be shown on the corresponding html. The two-legged metrics show the covariance relationships between two variables. Please let me know if you would this to be exactly the variable names. Thank you!